script.js

```diff
@@ -1,4 +1,71 @@
-// -----wheel-spin-js------
+var og = [
+  { label: '1x Carrot Juice'},
+  { label: '1x Cup of water'},
+  { label: '1x Vodka Redbull'},
+  { label: '2x Tuborg Gold'},
+  { label: '1x ScrollBar Punch'},
+  { label: '4x Pure Shots'},
+  { label: '2x Somersby'},
+  { label: '1x Pitcher'},
+  { label: '5x Tequilla shots'},
+  { label: '1x Draught Beer'},
+  { label: '3x Jägerbombs'},
+  { label: '2x Corona'},
+];
+
+var fileUpdate = document.getElementById('wheelInput');
+
+fileUpdate.addEventListener('change', function (e) {
+  const file = e.target.files[0];
+  if (file) {
+    const reader = new FileReader();
+    reader.onload = function (e) {
+      const tmp = e.target.result.split('\n');
+      data = tmp.map((item) => {
+        return { label: item.trim() };
+      }).filter((item) => item.label.length > 0);

```

Comment:
Should we add a validation here to ensure that the input matches our expected format, i.e. 12 items one per line?
Then send an alert("malformed") if not.

Reply:
I have added check